### ISSUE TITLE
🛠️ `Neighborhood`: Consider temporarily using a session recorder to learn how Space Owners are using Convene

### ISSUE THREAD

**Issue by anaulin** [zinc-collective/convene#1279]
As we start having Real Users, it would be nice to use some "session recording" software to help us understand how Convene is being used, so that we can improve the app to serve them better. Some options that I'm familiar with are:
https://www.fullstory.com/
https://www.hotjar.com/

These tools are a bit creepy, and we probably don't want to really depend on them in the mid-term. But maybe we can use them:
- for a limited time (say, collect data for ~1 month)
- only for space owners

We probably only need use the free tier of these tools.

**Comment by zspencer** [comment]
I agree that it would be helpful for us to gather this as we improve the product, and agree that limitin this to Space Owners within their own Space is probably the "SafeWay"

**Comment by anaulin** [comment]
I added a basic Hotjar installation (no identification or users or any other extra configuration) and it is now deployed. The Hotjar "installation verification" thing is happy.

I tried it on myself, and I see this in the console:
![image](https://user-images.githubusercontent.com/6729309/229385524-2e76bdd0-6473-4763-b561-99479199c866.png)

So we'll see how much information we actually end up receiving -- if our user base has ad blockers and other such tools, we might get very little info.
